lib/season_statable.rb

```diff
@@ -42,43 +42,23 @@ def worst_coach(season)
     min_win_percentage = coach_loss_percentage.values.min
     coach_loss_percentage.key(min_win_percentage)
   end
-  
-  def most_accurate_team(season)
-    all_season_game_id = @games.map do |game|
-      game.game_id if game.season == season
-    end.compact
 
-    team_id_goals_shots = @game_teams.each_with_object(Hash.new([0,0])) do |game, hash|
-      if all_season_game_id.include?(game.game_id)
-        hash[game.team_id] = [game.goals + hash[game.team_id][0], game.shots + hash[game.team_id][1]]
-      end
-    end
-    
-    avg_goals_made = team_id_goals_shots.transform_values do |value|
-      (value[0] / value[1].to_f).round(4)
-    end
+  def avg_goals_made(season)
```

Comment:
solid. Yeah we had some repitition with all_season_game_id so thanks for pulling that out as well. good stuff.